services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -204,7 +204,9 @@ async def _get_project_node_names(
     except DBProjectNotFoundError:
         _logger.exception("Could not find project: %s", f"{project_id=}")
     except ProjectNotFoundError as exc:
-        _logger.exception("Could not find parent project: %s", f"{exc.project_id=}")
+        _logger.exception(
+            "Could not find parent project: %s", exc.error_context()["project_id"]
```

Comment:
TIP: I would highly recommend using `create_troubleshotting_log_kwargs`
You can find a usage example in https://github.com/itisfoundation/osparc-simcore/blob/cd6f5455ab1d0e9ae874842926dab86100df07c4/services/api-server/src/simcore_service_api_server/exceptions/handlers/_handlers_factory.py#L34-L49

Reply:
only if this does not create a huge mess when I upgrade from master afterwards